build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
         jcenter()
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.1.4'
+        classpath 'com.android.tools.build:gradle:3.6.3'
```

Comment:
should this version bump be noted in the CHANGELOG?

Reply:
Yeah definitely